Updated French Strings - AppResourcesfr

CamilleIFC opened the pull request:
Fixed typos in certain values
Added accents in some values
Fixed odd or convoluted phrasing in some values
Chose more appropriate vocabulary in specific cases
Improved conjugation consistency

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/infiniteflight/infiniteflight-localization?pullRequest=143) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/infiniteflight/infiniteflight-localization?pullRequest=143) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/infiniteflight/infiniteflight-localization?pullRequest=143) it.</sub>

CamilleIFC commented:
Fixed typos in certain values
Added accents in some values
Fixed odd or convoluted phrasing in some values
Chose more appropriate vocabulary in specific cases
Improved conjugation consistency

carmichaelalonso commented:
Merci!
